<?xml version="1.0"?>
<launch>
    <arg name="run_diag" default="true"/>

    <node if="$(arg run_diag)" pkg="roboclaw_node" type="roboclaw_node_code.py" name="roboclaw_node" output="screen">
        <rosparam command="load" file="$(find roboclaw_node)/config/roboclaw_config.yaml"/>
    </node>

    <node pkg="diagnostic_aggregator" type="aggregator_node" name="diagnostic_aggregator">
        <rosparam command="load" file="$(find roboclaw_node)/config/roboclaw_diag.yaml"/>
    </node>

</launch>
